tables/automl/notebooks/retail_product_stockout_prediction/README.md

```diff
@@ -0,0 +1,371 @@
+  <tr>
+   <td>Stockout
+   </td>
+   <td>INTEGER
+   </td>
+   <td>Label 
+   </td>
+   <td>(1 - Stock-out, 0 - No stock-out)
+<p>
+When the demand for four weeks future sales is not met by the inventory in stock we say we see a stock-out. This is because an early warning sign would help the retailer re-stock inventory with a lead time for the stock to be replenished.
+   </td>
+  </tr>
+</table>
+
+
+To use AutoML Tables with BigQuery you do not need to download this dataset. However, if you would like to use AutoML Tables with GCS you may want to download this dataset and upload it into your GCP Project storage bucket. 
+
+Instructions to download dataset: 
+
+Sample Dataset: Download this dataset which contains sales data.
+
+1. [Link to training data](https://pantheon.corp.google.com/bigquery?folder=&organizationId=&project=product-stockout&p=product-stockout&d=product_stockout&t=stockout&page=table):  \
+Dataset URI: <bq://product-stockout.product_stockout.stockout>
+2. [Link to data for batch predictions](https://pantheon.corp.google.com/bigquery?folder=&organizationId=&project=product-stockout&p=product-stockout&d=product_stockout&t=batch_prediction_inputs&page=table):  \
+Dataset URI: <bq://product-stockout.product_stockout.batch_prediction_inputs>
+
+Upload this dataset to GCS or BigQuery (optional). 
+
+You could select either [GCS](https://cloud.google.com/storage/) or [BigQuery](https://cloud.google.com/bigquery/) as the location of your choice to store the data for this challenge. 
+
+1. Storing data on GCS: [Creating storage buckets, Uploading data to storage buckets](https://cloud.google.com/storage/docs/creating-buckets)
+2. Storing data on BigQuery: [Create and load data to BigQuery](https://cloud.google.com/bigquery/docs/quickstarts/quickstart-web-ui) (optional)
+
+
+## Building the model on AutoML Tables UI
+
+1. Enable [AutoML Tables](https://cloud.google.com/automl-tables/docs/quickstart#before_you_begin) on GCP. 
+
+2. Visit the [AutoML Tables UI](https://console.cloud.google.com/automl-tables) to begin the process of creating your dataset and training your model. \
+<img src="https://storage.cloud.google.com/cloud-ml-data/automl-tables/notebooks/automl_stockout_img/Image%201%202019-03-13%20at%201.02.53%20PM.png" width="50%" height="50%">\
+   
+3. Import your dataset or the dataset you downloaded in the last section \
+Click <+New Dataset> → Dataset Name <StockOut> → Click Create Dataset \
+<img src="https://storage.cloud.google.com/cloud-ml-data/automl-tables/notebooks/automl_stockout_img/Image%202%202019-03-13%20at%201.05.17%20PM.png" width="50%" height="50%">\
+
+4. You can import data from BigQuery or GCS bucket \
+    a. For BigQuery enter your GCP project ID, Dataset ID and Table ID \
+    After specifying dataset click import dataset \
+<img src="https://storage.cloud.google.com/cloud-ml-data/automl-tables/notebooks/automl_stockout_img/Image%203%202019-03-13%20at%201.08.44%20PM.png" width="50%" height="50%">\
```

Comment:
remove `< img src="` and replace with `![alt text](` 
and replace  `width="50%"` height="50%">` with `)`